### DISCUSSION TITLE
LLCC68 supported?

### DISCUSSION THREAD

**Discussion by aster1cs** [sandeepmistry/arduino-LoRa#651] [Q&A] [answered]
Hello and thanks for this great library!!

I am working on a LoRa project and have successfully used an ESP32 in combination with Ra-02 so far (they seem to use SX127X) [1].
Now I have bought a few Ra-01SC modules from aliexpress and LoRa.begin() fails on those even in the standard LoRaDumpRegisters-example using the same exact setup & wiring which works with the Ra-02. I connected 3xGND, 3V3, SCK, MOSI, MISO, NSS, RST and TXEN. I also passed the pins used to SPI and LoRa:
```
SPI.begin(14, 12, 13, 15);
LoRa.setPins(15, 18, 19);
if (!LoRa.begin(433E6)) {
    Serial.println("LoRa init failed. Check your connections.");
    while (true);                   // if failed, do nothing
  }
```

I tried to no avail:
- Reducing the SPI clock via setSPIFrequency()
- Soldering additional SMD caps (2x10µF + 1x0.1µF) directly onto the module
- Using another module of the same batch
- Using a different µC.
- Using different power sources

Then I digged deeper into the differences of the modules and realized that the Ra-01SC seems to use LLCC68 [1] and therefore might not be compatible with this library, as LLCC68 is not listed under Compatible Hardware in the readme. 🤦‍♂️
Is this the case? Is there no way to get the Ra-01SC modules running using this library or did I just miss something entirely else?

Thanks in advance.

> [1] https://www.linkedin.com/pulse/how-do-ra-01ra-01sra-01sc-sx127xsx126xllcc68-series-lora-module

**Comment by Kongduino** [comment, marked as answer]
It has indeed an LLCC68 inside, which means it's not compatible with the SX1276/8 chip. You can't use it with this library.

**Reply by aster1cs** [reply]
Thanks a lot for the confirmation. That's a pity, but it's my mistake. 😛

**Reply by udayalawa** [reply]
okay, but what library are you using now for ra01sc?

**Reply by aster1cs** [reply]
Since my project was based on the sandeepmistry-library and I didn't want to go down the rabbit hole of finding/evaluating another LoRa library suitable for Ra-01SC and then adapting my project to that library, I decided that for me the easier path was to change my hardware back to Ra-02.

While it seems that some libraries compatible to Ra-01SC do exist (like [nopnop2002](https://github.com/nopnop2002/Arduino-LoRa-Ra01S) or [chandrawi](https://github.com/chandrawi/LoRaRF-Arduino) ) I am not experienced with them.
It sure would be nice to hear from some more involved people about their experiences and recommendations with the Ra-01SC.
